README.md

```diff
@@ -377,6 +377,34 @@ okta:
     rateLimit:
       maxRetries: 4
 ```
+
+When you use OAuth 2.0 the full YAML configuration looks like:
+
+```yaml
+okta:
+  client:
+    connectionTimeout: 30 # seconds
+    orgUrl: "https://{yourOktaDomain}" # i.e. https://dev-123456.oktapreview.com
+    proxy:
+      port: null
+      host: null
+      username: null
+      password: null
+    authenticationScheme: "OAUTH2"
+    authorizationMode: "PrivateKey"
+    clientId: "{clientId}"
+    scopes:
+    - scope1
+    - scope2
+    privateKey:
+        # PEM format is recommended.
+        # This SDK supports following RSA AND EC algorithms - RS256, RS384, RS512, ES256, ES384, ES512.
+        keyFilePath: "~/.okta/privateKey.pem"
+        algorithm: "RS256"
```

Comment:
is this the default value?

Reply:
No, we don't recommend a default here AFAIK.

Reply:
Can we detect it for the PEM file?

Reply:
You mean detect the algorithm with the supplied pem file?

Reply:
Agreed on the better usability but it would be an overhead to detect the algorithm used (RSA-256 or EC-384 so on) with the .pem file (additional library might need to be pulled in).

Reply:
you shouldn't need to, see my previous comment above